.openpublishing.redirection.json

```diff
@@ -374,6 +374,136 @@
             "redirect_url": "/powershell/scripting/powershell-scripting",
             "redirect_document_id": "False"
         },
+        {
+            "source_path": "reference/6/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Add-Content-for-FileSystem.md",
+            "redirect_url": "reference/6/Microsoft.PowerShell.Core/Providers/about_FileSystem_Provider.md",
```

Comment:
The redirect_url is a URL path not a file path. source_path is a file path. The correct version of this entry should be:

```json
{ 
  "source_path": "reference/6/Microsoft.PowerShell.Core/Providers/FileSystem-Provider/Add-Content-for-FileSystem.md", 
  "redirect_url": "/powershell/module/Microsoft.PowerShell.Core/Providers/about_FileSystem_Provider", 
  "redirect_document_id": false 
},
```

You need to fix all of these entries.